hudi-common/src/main/java/org/apache/hudi/common/model/WriteOperationType.java

```diff
@@ -69,4 +69,8 @@ public static WriteOperationType fromValue(String value) {
         throw new HoodieException("Invalid value of Type.");
     }
   }
+
+  public static boolean isUpsert(WriteOperationType operationType) {
```

Comment:
rename `isChangingRecords()`  or `isChangeOperation()` ?

Reply:
Renamed to isChangingRecords.